networkx/generators/line.py

```diff
@@ -286,9 +286,6 @@ def inverse_line_graph(G):
         for u in p:
             P_count[u] += 1
 
-    if max(P_count.values()) > 2:
```

Comment:
I do not believe that this branch can be reached due to the fact that later functions force any vertex to be in at most 2 cliques